source/agora/common/Config.d

```diff
@@ -157,13 +157,13 @@ public struct NodeConfig
     public ushort stats_listening_port;
 
     /// Time to wait between request retries
-    public Duration retry_delay = 3.seconds;
+    public Duration retry_delay = 500.msecs;
 
     /// Maximum number of retries to issue before a request is considered failed
-    public size_t max_retries = 5;
+    public size_t max_retries = 10;
 
     /// Timeout for each request
-    public Duration timeout = 500.msecs;
+    public Duration timeout = 5000.msecs;
```

Comment:
Those should indeed be consistent with the example doc, see my comments there

Reply:
I changed the value to `5000.msecs` in all the related files. Am I missing something?

Reply:
So I assume we want 5 seconds for both `retry_delay` and `timeout`.

Reply:
OK, I got it. I'm applying it now. Ty.